pkgs/development/python-modules/naked/default.nix

```
@@ -0,0 +1,57 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, unittestCheckHook
+, requests
+, nodejs
+, ruby
+, nix-update-script
+}:
+
+buildPythonPackage rec {
+  pname = "naked";
+
+  version = "0.1.32";
+
+  src = fetchFromGitHub {
+    owner = "chrissimpkins";
+    repo = "naked";
+    rev = "v${version}";
+    hash = "sha256-KhygnURFggvUTR9wwWtORtfQES8ANd5sIaCONvIhfRM=";
+  };
+
+  postPatch = ''
+  rm tests/test_NETWORK* -rf # no internet in sandbox
+
+  # TODO: disable only specific broken tests
+  rm tests/test_SYSTEM.py
+  rm tests/test_TYPES.py
+  rm tests/test_SHELL.py
+
+  # fix hardcoded absolute paths
+  substituteInPlace **/*.* \
+    --replace /Users/ces/Desktop/code/naked /build/source
+  '';
+
+  # doCheck = false;
+
+  preCheck =''
+    cd tests
+
+    PATH=$PATH:$out/bin
+  '';
+
+  propagatedBuildInputs = [ requests ];
```

```suggestion
  propagatedBuildInputs = [
    requests
    pyyaml
  ];
```
https://github.com/chrissimpkins/naked/blob/ec4dd7af6c460a6f3c9835fa3a49bb4067045edc/setup.py#L46